Cargo.toml

```diff
@@ -13,45 +16,49 @@ lto = true
 opt-level = 3
 
 [dependencies]
-stremio-core = { git = "https://github.com/Stremio/stremio-core", branch = "development", features = ["env-future-send"] }
-stremio-derive = { git = "https://github.com/Stremio/stremio-core", branch = "development" }
-stremio-analytics = { git = "https://github.com/Stremio/stremio-core", branch = "development" }
+stremio-core = { git = "https://github.com/Stremio/stremio-core", branch = "development", features = [
+    "derive",
+    "analytics",
+    "env-future-send",
+] }
 stremio-watched-bitfield = { git = "https://github.com/Stremio/stremio-core", branch = "development" }
-serde = "1.0.*"
-serde_json = "1.0.*"
-futures = "0.3.*"
-http = "0.2.*"
-url = { version = "2.2.*", features = ["serde"] }
-percent-encoding = "2.1.*"
+serde = "1.0"
+serde_json = "1.0"
+futures = "0.3"
+http = "0.2"
+url = { version = "2.4", features = ["serde"] }
+percent-encoding = "2.1"
 chrono = "0.4.*"
 semver = { version = "1", features = ["serde"] }
-base64 = "0.13.*"
-hex = "0.4.*"
-flate2 = "1.0.*"
-either = "1.6.*"
-lazy_static = "1.4.*"
-enclose = "1.1.*"
-itertools = "0.10.*"
-strum = "0.22.*"
-strum_macros = "0.22.*"
-boolinator = "2.4.*"
-Inflector = "0.11.*"
-getrandom = "0.2.*"
-cfg-if = "0.1.*"
-auto_impl = "0.5.*"
-serde_path_to_error = "0.1.*"
-jni = "0.19.*"
+hex = "0.4"
+flate2 = "1.0"
+either = "1.6"
+lazy_static = "1.4"
+enclose = "1.1"
+itertools = "0.11"
+strum = { version = "0.25", features = ["derive"] }
+boolinator = "2.4"
+Inflector = "0.11"
+getrandom = "0.2"
+cfg-if = "1"
+serde_path_to_error = "0.1"
+# TODO: Updating the dependency requires changes to jni_ext
+jni = "0.19"
 reqwest = { version = "0.11", features = ["json", "rustls-tls"] }
-tokio = { version = "1.12.*", features = ["rt", "rt-multi-thread"] }
-openssl = { version = "0.10.*", features = ["vendored"] }
-bytes = "1.1.0"
-prost = "0.10.4"
-prost-types = "0.11.9"
-http-cache-reqwest = "0.4.5"
-reqwest-middleware = "0.1.6"
-once_cell = "1.12.0"
+tokio = { version = "1", features = ["rt", "rt-multi-thread"] }
+openssl = { version = "0.10", features = ["vendored"] }
+prost = "0.11"
+prost-types = "0.11"
+http-cache-reqwest = "0.11"
+reqwest-middleware = "0.2"
+once_cell = "1"
 
 [build-dependencies]
-prost-build = "0.11.9"
-protox = "0.3.3"
+prost-build = "0.11"
+protox = "0.4"
 glob = "0.3.0"
+
+# [patch.'https://github.com/Stremio/stremio-core']
+
+# stremio-core = { path = "../core" }
+# stremio-watched-bitfield = { path = "../core/stremio-watched-bitfield" }
```

Comment:
Can this be removed or is it some WIP left-overs?

Reply:
This is not a WIP. When developing or updating core it's easier to uncomment these patches and use your path core changes instead for a branch. We've done this for `core-web` as well.